README.md

```diff
@@ -90,6 +90,37 @@ The `optimize` operator supports any data structures and types. Serialize whatev
 
 Cloud providers such as [AWS](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), [Google Cloud](https://cloud.google.com/storage/docs/uploading-objects?hl=en#upload-object-cli), [Azure](https://learn.microsoft.com/en-us/azure/import-export/storage-import-export-data-to-files?tabs=azure-portal-preview) provide command line clients to upload your data to their storage solutions.
 
+If you're using [MinIO](https://min.io/), an S3-compatible object storage server, 
```

Comment:
Let's put this at the end of key features. This is bloating the explanation.

Reply:
And let s put the link to your repo too

Reply:
Sure